### ISSUE TITLE
pip installable Django app

### ISSUE THREAD

**Issue by zoidyzoidzoid** [acidjunk/crispy-forms-semanticUI-templates#2]
The most important part is making sure we package all the templates like `django-crispy-forms` does: https://github.com/maraujop/django-crispy-forms/blob/dev/MANIFEST.in , and whatever else is needed to have it work from adding it to `INSTALLED_APPS`. I assume it'll just be moving it into `crispy_semantic_ui/templates` or something.

It might be worth changing the template folder to `semantic_ui` or something so it can match the Django app name, though `crispy-semantic_ui` or something similar might be more specific.

P.S.: I have a [crispy form test project](https://github.com/zoidbergwill/crispy-test-project) that I used for testing my templates when I worked on the Bootstrap 4 stuff. If I make the time to work on this, I'll add it there too.

**Comment by acidjunk** [comment]
Will try the test project next week; when all form elements work ok; I can create a pip installable package.
Thanks for your help so far.

**Comment by zoidyzoidzoid** [comment]
Awesome. My pleasure.

Let me know once you're done, and I can help with packaging or anything else, if you like.

**Comment by rodmanning** [comment]
Hey, thanks for the version that's now on PyPi/pip. One problem: install from pip fails due to not being able to locate README_PIP.md:

```
(env) rod@T440 ~/src/git/deltoid-taslytic.git/env/lib $ pip install crispy-forms-semantic-ui
Collecting crispy-forms-semantic-ui
  Using cached crispy-forms-semantic-ui-0.1.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-ihcgbwjy/crispy-forms-semantic-ui/setup.py", line 6, in <module>
        with open(os.path.join(os.path.dirname(__file__), 'README_PIP.md')) as readme:
    FileNotFoundError: [Errno 2] No such file or directory: '/tmp/pip-build-ihcgbwjy/crispy-forms-semantic-ui/README_PIP.md'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-ihcgbwjy/crispy-forms-semantic-ui/
```

Problem appears to be that when running setup.py on the downloaded package, the following lines in `setup.py` require `README_PIP.md`:

```
with open(os.path.join(os.path.dirname(__file__), 'README_PIP.md')) as readme:
    README = readme.read()
```

However the contents of the tar that's created by `python setup.py bdist_egg` appear **not** to include any of your *.md files (or of the other text files in the top-level of the package):

```
$ tar --list -f dist/crispy-forms-semantic-ui-0.1.0.linux-x86_64.tar.gz

/env/lib/python3.5/
/env/lib/python3.5/site-packages/
/env/lib/python3.5/site-packages/crispy_forms_semantic_ui/
/env/lib/python3.5/site-packages/crispy_forms_semantic_ui/apps.py
/env/lib/python3.5/site-packages/crispy_forms_semantic_ui/templates/
/env/lib/python3.5/site-packages/crispy_forms_semantic_ui/templates/semantic-ui/
/env/lib/python3.5/site-packages/crispy_forms_semantic_ui/templates/semantic-ui/layout/
/env/lib/python3.5/site-packages/crispy_forms_semantic_ui/templates/semantic-ui/layout/baseinput.html
/env/lib/python3.5/site-packages/crispy_forms_semantic_ui/templates/semantic-ui/layout/multifield.html
/env/lib/python3.5/site-packages/crispy_forms_semantic_ui/templates/semantic-ui/layout/fieldset.html
/env/lib/python3.5/site-packages/crispy_forms_semantic_ui/templates/semantic-ui/layout/buttonholder.html
/env/lib/python3.5/site-packages/crispy_forms_semantic_ui/templates/semantic-ui/layout/div.html
/env/lib/python3.5/site-packages/crispy_forms_semantic_ui/templates/semantic-ui/field.html
/env/lib/python3.5/site-packages/crispy_forms_semantic_ui/templates/semantic-ui/betterform.html
/env/lib/python3.5/site-packages/crispy_forms_semantic_ui/templates/semantic-ui/errors_formset.html
/env/lib/python3.5/site-packages/crispy_forms_semantic_ui/templates/semantic-ui/multifield.html
/env/lib/python3.5/site-packages/crispy_forms_semantic_ui/templates/semantic-ui/display_form.html
/env/lib/python3.5/site-packages/crispy_forms_semantic_ui/templates/semantic-ui/errors.html
/env/lib/python3.5/site-packages/crispy_forms_semantic_ui/templates/semantic-ui/field.strict.html
/env/lib/python3.5/site-packages/crispy_forms_semantic_ui/templates/semantic-ui/whole_uni_form.html
/env/lib/python3.5/site-packages/crispy_forms_semantic_ui/templates/semantic-ui/uni_formset.html
/env/lib/python3.5/site-packages/crispy_forms_semantic_ui/templates/semantic-ui/uni_form.html
/env/lib/python3.5/site-packages/crispy_forms_semantic_ui/templates/semantic-ui/whole_uni_formset.html
/env/lib/python3.5/site-packages/crispy_forms_semantic_ui/__init__.py
/env/lib/python3.5/site-packages/crispy_forms_semantic_ui/__pycache__/
/env/lib/python3.5/site-packages/crispy_forms_semantic_ui/__pycache__/apps.cpython-35.pyc
/env/lib/python3.5/site-packages/crispy_forms_semantic_ui/__pycache__/__init__.cpython-35.pyc
/env/lib/python3.5/site-packages/crispy_forms_semantic_ui-0.1.0-py3.5.egg-info/
/env/lib/python3.5/site-packages/crispy_forms_semantic_ui-0.1.0-py3.5.egg-info/SOURCES.txt
/env/lib/python3.5/site-packages/crispy_forms_semantic_ui-0.1.0-py3.5.egg-info/requires.txt
/env/lib/python3.5/site-packages/crispy_forms_semantic_ui-0.1.0-py3.5.egg-info/dependency_links.txt
/env/lib/python3.5/site-packages/crispy_forms_semantic_ui-0.1.0-py3.5.egg-info/PKG-INFO
/env/lib/python3.5/site-packages/crispy_forms_semantic_ui-0.1.0-py3.5.egg-info/top_level.txt
```

Could just add these to the MANIFEST.in, but appears to be a related [issue on stackoverflow](http://stackoverflow.com/a/32776617), whereby non-package files in the top-level may not be included in the egg/tar archive unless the source dist is built (`python setup.py sdist`).

Will try to send a pull request later tonight with a fix.

**Comment by acidjunk** [comment]
I released a 0.2 version on pypi: I kept the APPNAME: crispy-forms-semantic-ui -> as it seems more consistent with django-crispty-forms itself.

For 0.3 I will start working on crispy-form Layout/Helper support and better docs.

Thanks for your help :)
